rpc/core/events.go

```diff
@@ -37,11 +33,13 @@ func Subscribe(ctx *rpctypes.Context, query string) (*ctypes.ResultSubscribe, er
 	subCtx, cancel := context.WithTimeout(ctx.Context(), SubscribeTimeout)
 	defer cancel()
 
-	sub, err := env.EventBus.Subscribe(subCtx, addr, q, subBufferSize)
+	sub, err := env.EventBus.Subscribe(subCtx, addr, q, env.Config.SubscriptionBufferSize)
 	if err != nil {
 		return nil, err
 	}
 
+	closeIfSlow := env.Config.CloseOnSlowClient
```

Comment:
Is this change necessary? I see the point, but it changes the semantics of the client in subtle ways. Did your experiments require this to be set to mitigate the disconnects? If not, could we reasonably defer this feature for now?

Reply:
The default behaviour here doesn't change at all (apart from the evaluation of the newly introduced `if` statement).

This change shows what I imagine the code should look like if we're really going to be closing the connection on slow clients. Until recently, when we inspected the code here, we didn't know that the node could just silently drop responses - something that could result in all kinds of nondeterministic behaviour for IBC.

So I'd say yes, this change is definitely necessary for us.